### ISSUE TITLE
Copy from not showed anymore ? 

### ISSUE THREAD

**Issue by Wewill** [qtranslate/qtranslate-xt#1055]
Since WP 5.8 or this is an isolated issue ? 

<img width="575" alt="Capture d’écran 2021-08-17 à 17 14 07" src="https://user-images.githubusercontent.com/3178443/129753004-9d5f4343-3b5c-47d3-bedd-df3f8886bcf7.png">

**Comment by herrvigg** [comment]
What do you expect in the element you showed? It doesn't seem related to qTranslate.

The "Copy from" button is close to the language switching buttons when you edit a post for example, it's still there. I'm aware there are some issues with this, it's not a high priority but this should be a different topic.

![image](https://user-images.githubusercontent.com/19155375/131217192-d9b23677-159a-4c96-9cfd-bf37cb52e7f7.png)

**Comment by Wewill** [comment]
@herrvigg You are right, I'm under Gutenberg, and I couldn't found this. My memory was not clear. 
But, in fact, this is relative to my issue. Under Gutenberg, you can't do nothing, and not copying from contents. 

When I create a Gutenberg post, even the languages are not set.

Can you create something quick from the post.php list ? 
Very fan of this plugin, but in fact, it is unusable with Gutenberg, unstable and unpredictable. 

Thanks a lot 
Best regards, 
Wilhem
